examples/go-client/access_workflow.go

```diff
@@ -5,14 +5,16 @@ import (
 	"log"
 	"time"
 
-	"github.com/gravitational/teleport/lib/auth"
-	"github.com/gravitational/teleport/lib/services"
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/lib/auth/client"
```

Comment:
It's a naming trade-off, I know. I'' accommodate for this wherever possible (e.g. where `lib/auth` is not also imported - otherwise, I'll have to rename either of them anyways).